modules/openapi-generator/src/main/java/org/openapitools/codegen/utils/ModelUtils.java

```diff
@@ -909,15 +911,19 @@ public static String getParentName(ComposedSchema composedSchema, Map<String, Sc
                         // discriminator.propertyName is used
                         return parentName;
                     } else {
-                        LOGGER.debug("Not a parent since discriminator.propertyName is not set {}", s.get$ref());
-                        // not a parent since discriminator.propertyName is not set
```

Comment:
Can we keep these comments so that people know the discriminator.propertyName in the spec is not set and therefore the model is not a "parent"?